packages/ts-codegen/src/__tests__/index.test.ts

```diff
@@ -744,6 +744,310 @@ describe(generateTS, () => {
     });
   });
 
+  it("import an entity, which is used to implement interface and returned by resolver", () => {
```

Comment:
Nit: the `it` is part of the sentence.

```suggestion
  it("imports an entity, which is used to implement interface and returned by resolver", () => {
```